include/micm/cuda/process/cuda_process_set.hpp

```diff
@@ -113,11 +113,28 @@ namespace micm
     micm::ProcessSet::SetJacobianFlatIds(matrix);
 
     ProcessSetParam hoststruct;
+    std::vector<ProcessInfoParam> jacobian_process_info_(this->jacobian_process_info_.size());
+    std::size_t i_process = 0;
+    for (const auto & process_info : this->jacobian_process_info_)
+    {
+      jacobian_process_info_[i_process].process_id_ = process_info.process_id_;
+      jacobian_process_info_[i_process].independent_id_ = process_info.independent_id_;
+      jacobian_process_info_[i_process].number_of_dependent_reactants_ = process_info.number_of_dependent_reactants_;
+      jacobian_process_info_[i_process].number_of_products_ = process_info.number_of_products_;
+    }
```

Comment:
shall we increase the `i_process` after each loop?

Reply:
Sorry, I got distracted and didn't realize I hadn't pushed my latest changes. This should be fixed now.